requirements-base.txt

```diff
@@ -35,7 +31,7 @@ Pillow>=6.2.2,<7.0.0
 progressbar2>=3.32,<3.33
 psycopg2-binary>=2.7.0,<2.9.0
 PyJWT>=1.5.0,<1.6.0
-pyrsistent==0.16.0 # TODO(py3): 0.17.0 requires python3, see https://github.com/tobgu/pyrsistent/issues/205
```

Comment:
This upgrade seems safe, most are formatting changes, CI changes and removal of six: https://github.com/tobgu/pyrsistent/compare/v0.16.0...v0.17.3